plugins/auth-backend/api-report.md

```diff
@@ -452,7 +452,7 @@ export const providers: Readonly<{
       signIn: {
         resolver: SignInResolver_2<CloudflareAccessResult>;
       };
-      cache?: CacheService | undefined;
+      cache?: CacheClient | undefined;
```

Comment:
How did this happen 🤔

Reply:
Smth smth API extractor smth type resolution smth, we have no idea. Got a ticket on the board to investigate but it's kinda low prio 😅